packages/api/server.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+require('dotenv').config();
+const sequelize = require('./helper/sequelize.js');
+const PORT = process.env.PORT || 8080;
```

Comment:
Since you're not currently using these, what do you think about removing them?

Reply:
removed